interfaces/apparmor/backend.go

```diff
@@ -304,8 +311,7 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 		return fmt.Errorf("cannot obtain expected security files for snap %q: %s", snapName, err)
 	}
 	dir := dirs.SnapAppArmorDir
-	glob1 := fmt.Sprintf("snap*.%s*", snapInfo.Name())
-	glob2 := fmt.Sprintf("snap-update-ns.%s", snapInfo.Name())
+	glob1, glob2 := profileGlobs(snapInfo.Name())
```

Comment:
can we agree that glob1, glob2 is very obscure? It was saved before by being right next to the glob itself, but now that it's separate I think it's worth finding a better name.

Or, continuing @pedronis's comment, you could have it return a `struct{ forsnap, forSun }` (but not actually `sun`), and then `glob := profileGlobs(...)` and then `glob.forSnap` etc.

Reply:
afaict they are used just to build a []string to be consumed by EnsureDirStateGlobs, that why my suggestion

Reply:
Not tested, how about something like this:
```
type profileGlobs strucft { snapName string }

// Globs returns a list of globs that describe apparmor profiles for a given snap.
func (pg profileGlobs) Globs() []string {
  return []string{...}
}
```

Reply:
I will just go for a `[]string` since that's easier and captures the intent better.